src/Explorer/index.jsx

```diff
@@ -1,68 +1,38 @@
 /* eslint no-underscore-dangle: 0 */
 /* eslint no-console: 0 */
 import React from 'react';
+import _ from 'lodash';
 import FilterGroup from '@gen3/ui-component/dist/components/filters/FilterGroup';
 import FilterList from '@gen3/ui-component/dist/components/filters/FilterList';
-import { askGuppyForAggregationData, getGQLFilter } from '@gen3/guppy/dist/components/Utils/queries';
+import { getGQLFilter } from '@gen3/guppy/dist/components/Utils/queries';
 import SummaryChartGroup from '@gen3/ui-component/dist/components/charts/SummaryChartGroup';
 import PercentageStackedBarChart from '@gen3/ui-component/dist/components/charts/PercentageStackedBarChart';
 import { config, components } from '../params';
 import ExplorerTable from './ExplorerTable/';
 import DataSummaryCardGroup from '../components/cards/DataSummaryCardGroup/.';
 import './Explorer.less';
-import { fetchWithCreds, fetchWithCredsAndTimeout, fetchUser } from '../actions';
-import { guppyDownloadUrl } from '../configs';
+import { fetchWithCredsAndTimeout, fetchUser } from '../actions';
+import { capitalizeFirstLetter } from '../utils';
 import { flatModelDownloadRelativePath, flatModelQueryRelativePath } from '../localconf';
 import getReduxStore from '../reduxStore';
 import Spinner from '../components/Spinner';
 import phenotypeNameMapping from './phenotypeNameMapping';
 
-const fieldMapping = config.dataExplorerConfig.fieldMapping;
-
-const fields = [];
-for (let j = 0; j < fieldMapping.length; j += 1) {
-  fields.push(fieldMapping[j].field);
-}
-const tableConfig = { fields };
-
-function addCountsToSectionList(filterSections) {
-  const filterSectionsCopy = filterSections.slice();
-  for (let k = 0; k < filterSectionsCopy.length; k += 1) {
-    const options = filterSectionsCopy[k].options.slice();
-    const n = Object.keys(options).length;
-    for (let m = 0; m < n; m += 1) {
-      options[m].count = 1;
-    }
-    filterSectionsCopy[k].options = options;
-  }
-  return filterSectionsCopy;
-}
-
-function calculateSummaryCounts(field, filteredData) {
-  const values = [];
-  for (let j = 0; j < filteredData.length; j += 1) {
-    values.push(filteredData[j][field]);
-  }
-  const uniqueValues = values.filter(
-    (value, index) => values.indexOf(value) === index,
-  );
-  return uniqueValues.length;
-}
-
 function checkIfFiltersApply(filtersApplied, row) {
   const attributes = Object.keys(filtersApplied);
   for (let i = 0; i < attributes.length; i += 1) {
     const property = attributes[i];
     if (!row[property]) {
       return false;
     }
-    const filtersApplyMatch = filtersApplied[property].selectedValues.map(
-      x => x.toLowerCase(),
-    ) === row[property].toLowerCase();
-    let filtersApplyContains = filtersApplied[property].selectedValues.filter(
-      x => row[property].toLowerCase() === x.toLowerCase(),
+    let value = row[property];
+    if (typeof value === 'object' && value.length === 1) {
```

Comment:
What's going on here? I didn't think object-type variables had a length property defined

Reply:
it's trying to flatten an 1-length array, so previously array like ["white"] and "white" won't match

Reply:
hmm i'm confused -- then shouldn't it be `if (Array.isArray(value) && value.length === 1) ` ?

Reply:
Oh I didn't know Array has this `isArray` function, will change to this. Thanks! :)